Maintenance/infrastructure/picasso.geometryfactory.com/reference_platforms/x64_Cygwin-Windows8_MSVC2013-Debug-32bits/CMakeCache.txt

```diff
@@ -897,7 +277,7 @@ WITH_CGAL_ImageIO:BOOL=ON
 WITH_CGAL_Qt3:BOOL=OFF
 
 //Enable CGAL component CGAL_Qt4
-WITH_CGAL_Qt4:BOOL=ON
+WITH_CGAL_Qt4:BOOL=OFF
```

Comment:
That will not work. That CMake cache is completely outdated. You must add a **new** variable `WITH_CGAL_Qt5` of type `BOOL` and value `OFF`.

If you edit the cache with an editor instead of with CMake-gui, then renamed that variable dfrom `WITH_CGAL_Qt4` to `WITH_CGAL_Qt5`.